build/common/installer/scripts/tomlparser-prom-customconfig.rb

```diff
@@ -223,7 +223,7 @@ def populateSettingValuesFromConfigMap(parsedConfig)
             File.open(file_name, "w") { |file| file.puts new_contents }
             puts "config::Successfully substituted the placeholders in telegraf conf file for replicaset"
             #Set environment variables for telemetry
-            file = File.open("telemetry_prom_config_env_var", "w")
+            file = File.open("prom_config_env_var", "w")
```

Comment:
@jatakiajanvi12 ,  as discussed, please list all the scenarios which you have tested to make sure we are not missing to validate any scenario.